### PR TITLE
[Spring Beans] LPS-203461

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang, I eliminated leftover bean usages and removed `messaging-misc-spring.xml` in this PR. 

https://liferay.atlassian.net/browse/LPS-203461

Thanks for reviewing,

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
@JanisZhang No regressions.

File: modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/liveusers/messaging/LiveUsersClusterBridgeMessageListener.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+/**
+ * @author Janis Zhang
+ */
+@Component(
+	property = "destination.name=" + DestinationNames.LIVE_USERS,
+	service = MessageListener.class
+)
+public class LiveUsersClusterBridgeMessageListener
+	extends ClusterBridgeMessageListener {
```
Comment (dantewang) [inline modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/liveusers/messaging/LiveUsersClusterBridgeMessageListener.java:24]: @JanisZhang It seems `ClusterBridgeMessageListener` is only used for the `live_users` destination, why not just move it?

File: modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/liveusers/messaging/LiveUsersMessageListener.java [hunk 2]
```diff
@@ -1,22 +1,28 @@
 /**
- * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.portal.liveusers.messaging;
+package com.liferay.site.admin.web.internal.liveusers.messaging;
```
Comment (dantewang) [inline modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/liveusers/messaging/LiveUsersMessageListener.java:6]: @JanisZhang Could you double check whether this should be in site.admin or user.admin modules?

**Review by dantewang — CHANGES_REQUESTED** [review]

**Pull request closed without merging**